applications/conceptual-model-editor/src/action/add-semantic-entities-to-visual-model.ts

```diff
@@ -147,11 +147,21 @@ async function addClassesAndClassProfilesToVisualModel(
 ) {
   for(const {entity, model, position} of validatedNodesData) {
     const modelIdentifier = model.getId();
-    if(isSemanticModelClass(entity)) {
-      await addSemanticClassToVisualModelAction(notifications, graph, classes, visualModel, diagram, entity.id, modelIdentifier, position);
+    // TODO RadStr: Hotfix for https://github.com/mff-uk/dataspecer/issues/1017 since it is called here,
+    //              so we catch the exception and move to next class
+    try {
+      if(isSemanticModelClass(entity)) {
+        await addSemanticClassToVisualModelAction(notifications, graph, classes, visualModel, diagram, entity.id, modelIdentifier, position);
+      }
+      else if(isSemanticModelClassProfile(entity)) {
+        await addSemanticClassProfileToVisualModelAction(notifications, graph, classes, visualModel, diagram, entity.id, modelIdentifier, position);
+      }
+      else {
+        notifications.error("Adding node of not supported type");
+      }
     }
-    else {
-      await addSemanticClassProfileToVisualModelAction(notifications, graph, classes, visualModel, diagram, entity.id, modelIdentifier, position);
+    catch {
+      // Do nothing
```

Comment:
There should probably be at least a log or something.
You should never silence an Exception.